utils/testUtils.js

```diff
@@ -1,29 +1,39 @@
 import { users } from 'models';
 import { init } from '../lib/testServer';
-export function mockDB() {
-    jest.doMock('models', () => {
-        const SequelizeMock = require('sequelize-mock');
-        const DBConnectionMock = new SequelizeMock();
-        const userMock = DBConnectionMock.define('users', {
-            id: 1,
-            firstName: 'Sharan',
-            lastName: 'Salian',
-            email: 'sharan@wednesday.is',
-            created_at: new Date(),
-            updated_at: new Date()
-        });
-        userMock.findByPk = query => userMock.findById(query);
-        return {
-            users: userMock
-        };
+
+export function configDB() {
+    const SequelizeMock = require('sequelize-mock');
+    const DBConnectionMock = new SequelizeMock();
+    const userMock = DBConnectionMock.define('users', {
+        id: 1,
```

Comment:
lets create a constant here with the user, and use that in both the test as well as here